packages/utils.jsx/spec/jsxBehaviors.ts

```diff
@@ -41,27 +41,36 @@ import {
 
 import { ORIGINAL_JSX_SYM } from '../dist/JsxObserver';
 
+import { assert } from "chai"
 
 class JsxTestObserver extends JsxObserver {
+
+  constructor (jsxOrObservable, parentNode, insertBefore = null, xmlns?, noInitialBinding?) {
+    super(jsxOrObservable, parentNode, insertBefore, xmlns, noInitialBinding)
+  }
+
   // For testing purposes, we make this synchronous.
   detachAndDispose (node) {
     super.detachAndDispose(node)
     cleanNode(node)
   }
+
+  dispose () {
```

Comment:
this is a no-op

Reply:
okay, but then we must change the import to `import { JsxObserver } from '../src'` and all types will be known (https://github.com/Auge19/tko/pull/4)